### PR TITLE
Alamo is immune to cas/OB

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

fixes: https://github.com/tgstation/TerraGov-Marine-Corps/issues/7420

## Why It's Good For The Game

It's just a delay tool, and can even if timed precisly kill the whole hive without them able to do anything

## Changelog
:cl:
balance: Alamo is immune to cas/OB
/:cl:

### CODE REVIEW

**Comment by QualityVan** [comment]
Does this make people in the alamo actually immune to cas/OB, or does it make the inside of the alamo itself not selectable as a primary target? If I drop HEOB in the fob right next to the alamo, will the people inside still die?

**Comment by BraveMole** [comment]
> Does this make people in the alamo actually immune to cas/OB, or does it make the inside of the alamo itself not selectable as a primary target? If I drop HEOB in the fob right next to the alamo, will the people inside still die?

Not sure. This makes alamo invulnerable to WP for sure, because walls are stopping it. 

HEOB probably bypass it.

**Comment by Ieteihowmee** [comment]
Boring. Pretty sure you can not target Alamo anymore anyways. Xenos mad. I do not think I have ever seen them get wiped on alamo with an OB. Hell I rarely see them DIE to an OB. OBs such massive balls, only worth is screening. Nothing stops them from just sprinting into alamo and somehow avoiding getting bombed. Makes no sense to me.

**Comment by KornFlaks** [comment]
Deep undergrounding the alamo proper wont do much except prevent direct lazes on the alamo.  CAS outside with high explosive such as a Fatty and you'll be able to reach inside with the explosives. This should stop the cheese of CASing while in flight hijack at least.

OBs on the other hand will remain largely unaffected. HE, incendary, and Cluster will recieve no effect with deep underground. I've been able to reach inside a deep underground location like the entrance of Ice Colony Labs by lazinng just outside with all three OB types. I expect Alamo to be no different.

So if you hear that BEEP BEEP. Run the fuck away.

**Comment by 2whatever2** [comment]
Ok but stopping lazes and shots in alamo is the intention in the first place though, if you took the time to even click on the issue the pr is fixing you'd see why.
image

**Comment by Ragecrabbo** [comment]
> 
> 
> Ok but stopping lazes and shots in alamo is the intention in the first place though, if you took the time to even click on the issue the pr is fixing you'd see why.
> image
> The problem is blasting the alamo while it tries to go up, xenos unable to get to safety at all. as leaving the dropship will kill them, and staying on will get them exploded.

clearly a skill issue

File: code/modules/shuttle/cas_plane.dm [hunk 1]
```diff
@@ -314,7 +314,7 @@
 		return
 	var/area/A = get_area(target)
 	if(A.ceiling >= CEILING_DEEP_UNDERGROUND)
-		to_chat(source, span_warning("That target is too deep underground!"))
+		to_chat(source, span_warning("Our payload won't reach this target!"))
```
Comment (TiviPlus) [inline code/modules/shuttle/cas_plane.dm:317]: this is lazy make it use an area flag instead of just changing the warning message

**Comment by BraveMole** [comment]
Tivi is lame

**Pull request closed without merging**